VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/confirmation/yes-no-popup-base.component.scss

```diff
@@ -12,9 +12,19 @@
 }
 .vh-popup {
   padding: 40px;
-  margin: 250px auto;
+  @media only screen and (min-width: 768px) {
+    margin: 250px auto;
+    width: 37%;
+  }
+  @media only screen and (min-width: 280px) and (max-width: 768px) {
+    margin: 100px auto;
+    width: 70%;
+  }
+  @media only screen and (min-height: 280px) and (max-height: 768px) {
+    margin: 10px auto;
+    width: 70%;
+  }
```

Comment:
Is there any need to provide a min-height and min-width on these? If we are thinking of this as our smallest size I would think we don't need a lower bound.

Reply:
yes, because the yes no popup position is fixed in the screen with popup-overlay, we need consider when the Judge close the hearing horizental scrren, the popup still in the center of the screen

Reply:
But why have a minimum bound defined at all? Can this not just be:

 @media only screen and (max-width: 768px) {
    margin: 100px auto;
    width: 70%;
  }
  @media only screen and (max-height: 768px) {
    margin: 10px auto;
    width: 70%;
  }